### PR TITLE
Enable i686 build for emulator

### PR DESCRIPTION
**Pull request by chunseoklee**
Now, i686 build support on tizen

Change-Id: Iaf1748e2a5929a03731d986cd2fb1c1b3cf7322d
Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

**Comment by chunseoklee** [comment]
will be cherrypicked into tizen 6.5 branch

File: infra/nnfw/cmake/buildtool/config/config_i686-tizen.cmake [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#
+# aarch64 tizen compile options
```
Comment (seanshpark) [inline infra/nnfw/cmake/buildtool/config/config_i686-tizen.cmake:2]: `aarch64 ` -> `i686`

File: infra/nnfw/cmake/buildtool/config/config_i686-tizen.cmake [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+#
+
+message(STATUS "Building for AARCH64 Tizen")
```
Comment (seanshpark) [inline infra/nnfw/cmake/buildtool/config/config_i686-tizen.cmake:5]: `AARCH64 ` -> `i686`

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by chunseoklee** [comment]
@nnfw-bot help

**Comment by chunseoklee** [comment]
@nnfw-bot test format-checker

**Comment by chunseoklee** [comment]
@nnfw-bot test nobody-checker

**Comment by chunseoklee** [comment]
@nnfw-bot test nobody-checker

**Pull request merged**